src/app/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Home() {
+  return <div>Home Page</div>;
```

Comment:
Nitpick - we prefer not to export an anonymous function. It’s just a readability thing, but it’s easier to do this as

const HomePage = () => {};

export default HomePage;